main/lsp/lsp_helpers.cc

```diff
@@ -236,6 +236,15 @@ string prettyTypeForMethod(const core::GlobalState &gs, core::SymbolRef method,
                        prettyDefForMethod(gs, method));
 }
 
+string prettyTypeForConstant(const core::GlobalState &gs, core::SymbolRef constant, core::TypePtr type) {
+    core::TypePtr result = type;
+    if (constant.data(gs)->isTypeAlias()) {
+        // By wrapping the type in `MetaType`, it displays as `<Type: Foo>` rather than `Foo`.
+        result = core::make_type<core::MetaType>(type);
+    }
+    return result->showWithMoreInfo(gs);
+}
```

Comment:
Honestly not sure what's the point of this snippet. I saw we were doing it for hover, so I decided to do it for completion too. Is it valuable? We could remove it.

Reply:
What snippet? The `MetaType` change?

Reply:
iirc it disambiguates from type `Foo` and type `<Type: Foo>`... but I'm willing to accept arguments for different behavior.